libs/blocks/icon-block/icon-block.css

```diff
@@ -289,14 +288,6 @@
   line-height: var(--type-body-m-lh);
 }
 
-.two-up .icon-block.fullwidth,
-.three-up .icon-block.fullwidth,
-.four-up .icon-block.fullwidth,
-.five-up .icon-block.fullwidth {
-  grid-column: 1 /-1;
-}
```

Comment:
Just checking was this meant to be removed?

Reply:
Yes in favor of that style being applied in the section grid (extending to all full width children)